pkg/printers/internalversion/printers_test.go

```diff
@@ -1094,12 +1095,13 @@ func TestPrintHunmanReadableIngressWithColumnLabels(t *testing.T) {
 		},
 	}
 	buff := bytes.NewBuffer([]byte{})
+	parsers := []*jsonpath.JSONPath{}
 	table, err := printers.NewTablePrinter().With(AddHandlers).PrintTable(&ingress, printers.PrintOptions{ColumnLabels: []string{"app_name"}})
 	if err != nil {
 		t.Fatal(err)
 	}
 	verifyTable(t, table)
-	if err := printers.PrintTable(table, buff, printers.PrintOptions{NoHeaders: true}); err != nil {
+	if err := printers.PrintTable(table, parsers, buff, printers.PrintOptions{NoHeaders: true}); err != nil {
```

Comment:
These will not be necessary when you apply the suggested change wrt decorating the table.